Schema/klant.subjectIdentificatie.schema.json

```diff
@@ -1,54 +1,80 @@
 {
-  "$id": "https://klantenBundle.commonground.nu/klant.natuurlijkPersoon.schema.json",
+  "$id": "https://klantenBundle.commonground.nu/klant.subjectIdentificatie.schema.json",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "version": "0.1",
-  "title": "NatuurlijkPersoon",
+  "title": "SubjectIdentificatie",
   "type": "object",
   "properties": {
     "inpBsn": {
       "type": "string",
       "description": "Het burgerservicenummer, bedoeld in artikel 1.1 van de Wet algemene bepalingen burgerservicenummer.",
-      "nullable": true
+      "nullable": true,
+      "maxLength": 9
     },
     "anpIdentificatie": {
       "type": "string",
       "description": "Het door de gemeente uitgegeven unieke nummer voor een ANDER NATUURLIJK PERSOON",
-      "nullable": true
+      "nullable": true,
+      "maxLength": 17
     },
     "inpANummer": {
       "type": "string",
       "description": "Het administratienummer van de persoon, bedoeld in de Wet BRP",
-      "nullable": true
+      "nullable": true,
+      "maxLength": 10
     },
     "geslachtsnaam": {
       "type": "string",
       "description": "De stam van de geslachtsnaam.",
-      "nullable": true
+      "nullable": true,
+      "maxLength": 200
     },
     "voorvoegselGeslachtsnaam": {
       "type": "string",
-      "nullable": true
+      "nullable": true,
+      "maxLength": 80
     },
     "voorletters": {
       "type": "string",
       "description": "De verzameling letters die gevormd wordt door de eerste letter van alle in volgorde voorkomende voornamen.",
       "nullable": true,
-      "format": "uri"
+      "maxLength": 20
     },
     "voornamen": {
       "type": "string",
       "description": "Voornamen bij de naam die de persoon wenst te voeren.",
-      "nullable": true
+      "nullable": true,
+      "maxLength": 200
     },
     "geslachtsaanduiding": {
       "type": "string",
       "description": "Een aanduiding die aangeeft of de persoon een man of een vrouw is, of dat het geslacht nog onbekend is.\n\nUitleg bij mogelijke waarden:\n\n* `m` - Man\n* `v` - Vrouw\n* `o` - Onbekend",
-      "nullable": true
+      "nullable": true,
+      "enum": [
+        "m",
+        "v",
+        "o"
+      ]
     },
     "geboortedatum": {
       "type": "string",
-      "description": "De achternaam van de klant.",
-      "nullable": true
+      "nullable": true,
+      "maxLength": 9
+    },
+    "vestigingsNummer": {
+      "type": "string",
+      "description": "Een korte unieke aanduiding van de Vestiging.",
+      "nullable": true,
+      "maxLength": 24
+    },
+    "handelsnaam": {
+      "type": "array",
+      "description": "De naam van de vestiging waaronder gehandeld wordt.",
+      "nullable": true,
+      "items": {
+        "type": "string",
+        "maxLength": 625
+      }
     },
     "verblijfsadres": {
       "$ref": "https://klantenBundle.commonground.nu/klant.verblijfAdres.schema.json"
```

Comment:
Git doet hier geen auto-formatting, dat betekent niet meteen dat er iets mis is, maar als je 2 seconden in PHPstorm kan checken of er geen syntax fout ergens is belandt....